vignettes/knit_print.Rmd

```diff
@@ -9,11 +9,11 @@ citation-location: margin
 bibliography: vignettes.bib
 vignette: >
   %\VignetteIndexEntry{demo}
-  %\VignetteEngine{quarto::html}
+  %\VignetteEngine{knitr::rmarkdown}
   %\VignetteEncoding{UTF-8}
 ---
 
-```{r}
+```{r, label = "setup"}
 #| label: setup
```

Comment:
Does this work in Rmd files? Or is just interpreted as a comment `#| echo: FALSE` ?

Reply:
Did this work?  well, it didn't fail! :-) To be honest, I simply couldn't remember the syntax and just guessed.  When it didnt fail, I just carried on.   Checking back, it should have been `{r, setup}`.